cluster/manifests/01-admission-control/routegroups-webhook.yaml

```diff
@@ -1,11 +1,11 @@
-{{ if eq .Cluster.ConfigItems.routegroups_validation "enabled" }}
 apiVersion: admissionregistration.k8s.io/v1
 kind: ValidatingWebhookConfiguration
 metadata:
-  name: "routegroup-admitter.teapot.zalan.do"
+  name: "skipper-admitter.teapot.zalan.do"
   labels:
-    application: routegroups-admission-webhook
+    application: skipper-admission-webhook
```

Comment:
Should we take a chnace and use the same application as for skipper ingress (and `webhook` component maybe)?

Reply:
I think that wouldn't be a problem.
FYI those application IDs are not registered.